pkg/products/threescale/reconciler.go

```diff
@@ -1828,21 +1828,21 @@ func (r *Reconciler) reconcile3scaleMultiTenancy(ctx context.Context, serverClie
 				r.log.Errorf("Failed to get KeycloakClient for tenant account",
 					l.Fields{
 						"tenantAccountId":    account.Id,
-						"tenantAccountName":  account.Name,
+						"tenantAccountName":  account.OrgName,
 						"tenantAccountState": account.State,
 					},
 					err,
 				)
 				continue
 			}
 
-			r.log.Infof("Checking Keycloak state", l.Fields{"tenantAccountName": account.Name})
+			r.log.Infof("Checking Keycloak state", l.Fields{"tenantAccountName": account.OrgName, "user" : keycloak.UserPhaseReconciled, "client":kcClient.Status.Ready})
```

Comment:
```suggestion
			r.log.Infof("Checking Keycloak state", l.Fields{"tenantAccountName": account.OrgName, "user": kcUser.Status.Phase, "client": kcClient.Status.Ready})
```
@briangallagher This needs to be changed to report kcUser's current status